FunctionalTableData/TableSectionHeaderFooter.swift

```diff
@@ -19,13 +19,7 @@ public protocol TableHeaderFooterConfigType: TableItemConfigType {
 	var height: CGFloat { get }
 }
 
-public extension TableHeaderFooterConfigType {
-	func isEqual(_ other: TableHeaderFooterConfigType?) -> Bool {
-		return (other as? Self) != nil
-	}
-}
```

Comment:
Removing this is potentially a breaking change for protocol adopters who relied on the default implementation. It should be enough to just override the method like you did below, while keeping this one here as the default.

Reply:
Breaking changes are fine, master is currently in a state of broken changes as it works towards its 2.0 release. If you need non-breaking changes then you stick to the tagged release

Reply:
Oh yeah, that's right. Though I still think we should keep the default implementation.

Reply:
> Oh yeah, that's right. Though I still think we should keep the default implementation.

I... disagree. The default one isn't great and we don't have a default for CellConfigType's isEqual

Reply:
Fair enough, no strong opinion.